### ISSUE TITLE
Plugin shows no response to any command

### ISSUE THREAD

**Issue by GregStef** [OlehL/cuda_plain_tasks#12]
Hello
First of all, this might not be an issue at all - I might be doing something wrong myself. So, my apologies in advance for reporting an issue, but I could not find any other way to contact you.

My box is Ubuntu 18.04.5 LTS, 4.15.0-118-generic, x86_64.
I installed CudaText just yesterday afternoon (trying to get rid of Sublime) and Plain Tasks (along with other plugins).
CudaText is 1.112.1.2, linux-x86_64-gtk2,fpc 3.2.1.

Today I tried to create a todo list, but all my attempts failed:
   Alt+i 
   Menu Plugins - Plain Tasks - New
   Command Palette - plugin:Plain Tasks:New
I tried: in an empty document, in a document containing a few lines and saved as 'test.todo' and, finally, to a "~/.config/cudatext/data/newdoc/default.todo" (which, by the way, does not appear as template in File - New from template).

Installation must be ok (I can see the readme file etc). I never saw any error or other message, both during installation and attempts.

While writing this, I saw cudatext_1.113.2.0-1_gtk2_amd64.deb was available - I installed and tried again, same results.

Any help would be greatly appreciated...

**Comment by Alexey-T** [comment]
@OlehL 
I tried to call "plugins / plain tasks / new" and got statusbar error text "plugin is only for lexer: ToDo". so you make requirement for "new" command - lexer ToDo. bad. "new" must work always?

after I removed line "lexers=" in plugin's install.inf, plugin makes new task. BUT didn't set lexer ToDo for new file!
i will try to fix.

**Comment by Alexey-T** [comment]
@GregStef try to install my fork with the fix:

-uninstall the plugin
-"Plugins/ Addon Manager/ Install from github", URL https://github.com/Alexey-T/cuda_plain_tasks

@OlehL Check fix in my repo, do the PullReq from there?

**Comment by Alexey-T** [comment]
@GregStef 
Also plugin needs the lexer ToDo (AddonManager installs it automatically), is this lexer present in lexers-menu?

**Comment by GregStef** [comment]
@Alexey-T 
TLTR: It works - a million thanks!

Uninstalling the plugin did not remove the lexer (don't know if it should), so I removed it manually (Options/ Lexers/ Lexer library/ delete).
I installed from your repo and, after restarting CudaText, it didn't seem to work.
An epiphany led me to check the lexer: It was missing, the "Install from github" process did not install it.
I installed it and the plugin worked.

Thanks again for helping - and for a superb piece of software!
